src/platforms/common/data-management/debug-files/upload/index.mdx

```diff
@@ -20,6 +20,15 @@ application:
 
 ![](debug-files-workflow.png)
 
+<PlatformSection supported={["dotnet"]}>
+<Note>
+
+The Sentry support for .NET requires uploading debug files only on mobile (Android and iOS) apps.
```

Comment:
This needs to be updated once https://github.com/getsentry/sentry-dotnet/issues/1740 is completed, cc @Swatinem, @ashwoods